TensorflowInf/tf-inference.sh

```diff
@@ -1,122 +1,7 @@
 set -e
 
-# Check for Python 3 version
-function python_check() {
```

Comment:
Maybe we can leave the python check? We do use the "PYTHON_CMD" variable below

Reply:
Do we need to use `PYTHON_CMD`? is there any reason why we can't replace that with `python3`?

Reply:
Great question.... no reason lmao. Looks nicer I guess 🤣 either way is fine honestly

Reply:
Actually do we want to check if the python version is between 3.5 and 3.7?

Reply:
Tbh I don't think we have to since the right versions of the requirements won't install without it. E.x. I have python 3.8 on my laptop and can't run this :) 
So if the user has the requirements then they should be fine

Reply:
Yepp same, doesn't in my default python environment either, had to use pyenv.

Reply:
Sounds good then, you can replace `PYTHON_CMD` with `python3` and then merge to master